pkg/freeze/containerd/pause_test.go

```diff
@@ -120,14 +135,23 @@ func TestContainerResume(t *testing.T) {
 }
 
 func TestNoQueueProxyPause(t *testing.T) {
+	queueProxy := Container("queueproxy", "queue-proxy")
+
 	containers = []*cri.Container{&queueProxy}
 	results = []string{}
+	method = ""
 
-	f := FakeContainerdCri{}
-	cntr, _ := f.List(nil, nil, "")
-	ids, _ := lookupContainerIDs(cntr)
-	if !compareContainerLists(ids, results) {
-		t.Errorf("expected %s, got %s", ids, results)
+	f := FakeContainerdCri{
+		containers: containers,
+		results:    results,
+		method:     method,
+	}
```

Comment:
I'd just put these in the struct initialiser since the local variables aren't used elsewhere (also I think the default value should be fine for results and method)

```suggestion
	f := FakeContainerdCri{
		containers: []*cri.Container{Container("queueproxy", "queue-proxy")},
	}
```